CMakeLists.txt

```diff
@@ -80,7 +80,7 @@ endif ()
 #in the utility libraries, that don't affect the data format itself.
 # For more information see http://semver.org/
 set(ISMRMRD_VERSION_MAJOR 1)
-set(ISMRMRD_VERSION_MINOR 5)
+set(ISMRMRD_VERSION_MINOR 6)
```

Comment:
This should be the PATCH version you update. The guidance is anything that messes with the binary layout of structures is a major version. Anything that messes with the flexible (XML) header is a minor version. And small tweaks, like this one, is PATCH. So it would be 1.5.1.